csharp/ql/src/Security Features/CWE-022/ZipSlip.ql

```diff
@@ -1,90 +1,20 @@
 /**
- * @name Potential ZipSlip vulnerability 
- * @description When extracting files from an archive, don't add archive item's path to the target file system path. Archive path can be relative and can lead to 
- * file system access outside of the expected file system target path, leading to malicious config changes and remote code execution via lay-and-wait technique 
+ * @name Arbitrary file write during zip extraction ("ZipSlip")
```

Comment:
"ZipSlip" > "Zip Slip"

(According to the linked reference: https://snyk.io/research/zip-slip-vulnerability)

Reply:
Done.